backend/de.metas.business/src/main/java/de/metas/pricing/rules/Discount.java

```diff
@@ -190,8 +188,6 @@ private static void updatePricingResultFromPricingConditionsResult(
 			return;
 		}
 
-		pricingResult.setDiscount(pricingConditionsResult.getDiscount());
```

Comment:
..i guess this is not what you wanted, right?

Reply:
right, corrected.